Makefile

```diff
@@ -27,6 +27,7 @@ install:
 	-rm -rf dist
 	python setup.py bdist_wheel
 	pip install --upgrade dist/*.whl
+	$(MAKE) bindings
```

Comment:
This is still not good. From a clean checkout, this will result in a broken installed package. Let's chat about this offline.